tonic/tonic.go

```diff
@@ -303,18 +303,28 @@ func extractPath(c *gin.Context, tag string) (string, []string, error) {
 	if err != nil {
 		return "", nil, err
 	}
-	p := c.Param(name)
+
+	var params []string
+
+	if !c.GetBool(ArrayTag) {
```

Comment:
OpenAPI 3 spec defines the path parameters serialization with `explode` being false by default. Do we want to follow that, and make comma-splitting an opt-in ?

Reply:
OpenAPI3 defines explode=false by default specifically.
Here, we assume a scalar.
I feel like an opt-in split mechanism, clearly defining the format as a comma-separated list, is clearer for both query & path handling.